README.md

```diff
@@ -1,24 +1,49 @@
 # cmake_example for pybind11
 
-[![Build Status](https://travis-ci.org/pybind/cmake_example.svg?branch=master)](https://travis-ci.org/pybind/cmake_example)
-[![Build status](https://ci.appveyor.com/api/projects/status/57nnxfm4subeug43/branch/master?svg=true)](https://ci.appveyor.com/project/dean0x7d/cmake-example/branch/master)
 
-An example [pybind11](https://github.com/pybind/pybind11) module built with a
-CMake-based build system. This is useful for C++ codebases that have an existing
-CMake project structure.
+|      CI              | status |
+|----------------------|--------|
+| Linux/macOS Travis   | [![Travis-CI][travis-badge]][travis-link] |
+| MSVC 2015            | [![AppVeyor][appveyor-badge]][appveyor-link] |
+| conda.recipe         | [![Conda Actions Status][actions-conda-badge]][actions-conda-link] |
+| pip builds           | [![Pip Actions Status][actions-pip-badge]][actions-pip-link] |
+| [`cibuildwheel`][]   | [![Wheels Actions Status][actions-wheels-badge]][actions-wheels-link] |
+
+[gitter-badge]:            https://badges.gitter.im/pybind/Lobby.svg
+[gitter-link]:             https://gitter.im/pybind/Lobby
```

Comment:
This are not used?

Reply:
I think I had them there for python_example, probably missed them here?

Reply:
Fixes by matching python_exampe.